From b9ceaf581aadf725dbedc7f2269495f1791c52d1 Mon Sep 17 00:00:00 2001
From: Hisashi Nakamura <hisashi.nakamura.ak@renesas.com>
Date: Tue, 11 Jun 2013 13:46:34 +0900
Subject: [PATCH 476/715] ARM: shmobile: lager: Change bootargs in DTS

The console baud rate is made to match mini monitor,
and NFS root filesystem is added.

Signed-off-by: Hisashi Nakamura <hisashi.nakamura.ak@renesas.com>
(cherry picked from commit b56d2e36df10f0d89dffa4ae7a29bb9b541438b9)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 arch/arm/boot/dts/r8a7790-lager.dts |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/r8a7790-lager.dts b/arch/arm/boot/dts/r8a7790-lager.dts
index 09a84fc..646ef5a 100644
--- a/arch/arm/boot/dts/r8a7790-lager.dts
+++ b/arch/arm/boot/dts/r8a7790-lager.dts
@@ -1,6 +1,7 @@
 /*
  * Device Tree Source for the Lager board
  *
+ * Copyright (C) 2013 Renesas Electronics Corporation
  * Copyright (C) 2013 Renesas Solutions Corp.
  *
  * This file is licensed under the terms of the GNU General Public License
@@ -16,7 +17,7 @@
 	compatible = "renesas,lager", "renesas,r8a7790";
 
 	chosen {
-		bootargs = "console=ttySC6,115200 ignore_loglevel";
+		bootargs = "console=ttySC6,38400 mem=768M ip=dhcp root=/dev/nfs ignore_loglevel";
 	};
 
 	memory@40000000 {
-- 
1.7.10.4

